pkg/volume/csi/csi_plugin.go

```diff
@@ -51,8 +51,6 @@ const (
 
 	// TODO (vladimirvivien) implement a more dynamic way to discover
 	// the unix domain socket path for each installed csi driver.
-	// TODO (vladimirvivien) would be nice to name socket with a .sock extension
-	// for consistency.
 	csiAddrTemplate = "/var/lib/kubelet/plugins/%v/csi.sock"
```

Comment:
I believe this is no longer used as well, that line can also be removed.

Reply:
Referring to the `csiAddrTemplate` variable here.

Reply:
hmm i'll give it a git-grep later :+1: